cmd/thv-operator/Taskfile.yml

```diff
@@ -123,22 +123,39 @@ tasks:
       - helm upgrade --install toolhive-operator deploy/charts/operator --namespace toolhive-system --create-namespace --kubeconfig kconfig.yaml
 
   operator-deploy-local:
-    desc: Build the ToolHive runtime and Operator image locally and deploy it to the K8s cluster
+    desc: |
+      Build the ToolHive runtime and Operator image locally and deploy it to the K8s cluster.
+      Set DEPLOY_REGISTRY_API=true to also build and deploy the registry API image.
```

Comment:
just thinking aloud: to enable the registry controller we use `ENABLE_EXPERIMENTAL_FEATURES=true`, shouldn't we have a single flag for these experimental features?